labs/LorumIpsum/LorumIpsum.md

```diff
@@ -0,0 +1,404 @@
+
+Sed dictum lectus quis eros tempor lacinia. Class aptent taciti
+sociosqu ad litora torquent per conubia nostra, per inceptos
+himenaeos. Sed venenatis ante neque, lobortis cursus erat bibendum
+et. Maecenas pharetra ullamcorper ipsum, vel faucibus velit tincidunt
+id. Class aptent taciti sociosqu ad litora torquent per conubia
+nostra, per inceptos himenaeos. Mauris ex augue, lacinia sed metus
+vel, imperdiet viverra dolor. Etiam id pretium lectus, ac convallis
+ante. Sed porta, turpis a commodo eleifend, metus elit vehicula nulla,
+quis malesuada leo ipsum non mi. Proin consectetur lacinia rhoncus. In
+eleifend fermentum mi at volutpat. Aliquam erat volutpat. Quisque a
+elit non augue malesuada volutpat a ultrices ligula.
+
+Sed ligula urna, gravida ac arcu eu, placerat dictum sem. Curabitur
+bibendum leo in orci ultrices, eget finibus purus dignissim. Praesent
+at magna non neque cursus sagittis non vel purus. Donec sit amet augue
+venenatis, auctor ante ullamcorper, tincidunt sapien. Proin ac
+convallis purus. Sed facilisis tellus in justo suscipit, at volutpat
+lectus lacinia. Vestibulum viverra, lorem ac tempor efficitur, dui
+turpis ultricies ligula, sed ultricies urna mi nec metus.
+
+
+# Algorithm Transition Mechanisms
+
+Curabitur vulputate accumsan augue, eget vulputate massa ornare
+eu. Fusce et diam in eros facilisis vehicula sit amet a quam. Nullam
+fermentum mauris eu augue rhoncus, interdum pulvinar ligula
+finibus. Pellentesque nisl sapien, molestie at tempor molestie,
+scelerisque id neque. Donec rhoncus id odio et pharetra. Mauris vel
+risus a leo tristique consequat. Vivamus vel ex in ex posuere
+tincidunt sit amet ac orci.
+
+Praesent tristique neque dui, quis vulputate nisl dictum at. Fusce sed
+egestas ante, et fringilla ligula. Vivamus laoreet porttitor
+imperdiet. Cras rutrum mauris in leo condimentum consequat. Quisque
+interdum posuere enim, eu laoreet nisi pulvinar in. Integer bibendum
+diam nec cursus malesuada. Duis vulputate consequat est, et
+condimentum nunc scelerisque ut.
+
+
+# Wrapped Key Tagging
+
+Etiam malesuada eu enim in semper. Pellentesque eget vehicula
+ligula. In aliquet massa posuere pellentesque porttitor. Mauris
+consequat enim nec velit cursus, eu scelerisque purus
+tincidunt. Quisque pulvinar enim urna, non ullamcorper est blandit
+et. Nam massa ante, molestie ac malesuada et, eleifend vitae lacus. Ut
+tempus neque mi, id ullamcorper ex eleifend sit amet.
+
+
+## Specification
+
+The 13-character constant ASCII string "LorumIpsumKey" followed by a
+given 3-ASCII-digit key issue number will be prepended to the 192-bit
+wrapped key after the first CRC-32c has been applied.
+
+
+### Valid Issue Numbers
+
+The KLI20 will accept any 24-bit issue number. However, this key
+specification issue only uses numbers that correspond to the ASCII
+strings between and including "000" to "999". Specifically, this key
+specification follows a special schedule that is comprised entirely of
```

Comment:
"comprised" --> "composed"  https://www.businesswritingblog.com/business_writing/2015/01/comprised-of-vs-composed-of-a-test.html